### ISSUE TITLE
Re-design the footer

### ISSUE THREAD

**Issue by SaumyaSubham** [swapagarwal/swag-for-dev#1274]
## Issue regarding the website

- [ ] I've checked that this isn't a duplicate issue.

Hii @swapagarwal,
I want to redesign the footer and adding social media icons also so that the users can communicate or keep in touch with you and your updates. Comment your social media handles or I just keep a dummy link and you just have to replace it with your own social media handles but make sure you mention your social media like twitter(X), Instagram, LinkedIn etc.

Kindly assign me this issue under hacktoberfest 2024
Thank You

**Comment by shaikmynuddin** [comment]
Yes same here from fest 24 I will keep fake link kindly just change it can you mention some structural design like measures of your footer

**Comment by rohitmore07** [comment]
Hey @swapagarwal I am very keen to work on this issue. I would be really happy if you could assign this issue to me.

**Comment by tushars-code** [comment]
can i work on this?
